pkg/aws/common.go

```diff
@@ -45,12 +44,34 @@ func ValidMasterReplicas() []int {
 	return []int{1, 3}
 }
 
+// ValidLabelAdmins returns the list of accounts used to manipulate labels
+func ValidLabelAdmins() []string {
+	return []string{
+		"system:serviceaccount:giantswarm:api",
```

Comment:
This is taken from `gatekeeper`. The username that is used by the `api` for label manipulation is hardcoded.